app/views/complaints/_monitoring_timeline_event.html.erb

```diff
@@ -0,0 +1,18 @@
+<h3 class="usa-process-list__heading ct-timeline__two-part-container">
```

Comment:
[thought] we could *probably* better align the monitoring/ttahub events and reduce duplication, but *also* probably not worth it

Reply:
I just got a couple design pointers from Laura which are going to make them diverge just slightly more, plus they're pulling different fields.....basically I decided it wasn't worth it for the small amount of code in these files